src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const STANDART_COAST = 40;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const MIDEL_TERM = 3;
+  const MIDEL_TERM_DISCOUNT = 20;
+
+  if (days < MIDEL_TERM) {
+    return days * STANDART_COAST;
```

Comment:
```suggestion
    return standartPrice;
```
consider creating a constant for repeating code. For example, ```const standartPrice = days * STANDART_COAST;```